test/functionalTest/cases/719_servicepath_for_registrations/cm_forwards_cb_with_servicePath.test

```diff
@@ -0,0 +1,456 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Just to synchronize ...

What about this test:

    #
    # 01. Register E1/A1 with service path /c1/c1/e1
    # 02. Register E2/A1 with service path /c1/c2/e2
    # 03. Register E3/A1 with service path /c1/c2/e3
    # 04. Register E4/A1 with service path /c2/c3/e4
    # 05. Register E5/A1 with service path /c2/c3/e4
    # 06. Register E6/A1 with service path /c2/c4/e4
    # 07. Query E.*/A1 with service path /c2/c3/#
    # 08. Query E.*/A1 with service path /c1/c1/e1,/c1/c2/e2
    # 09. Query E.*/A1 with service path /c1/#,/c2/c4/e4
    #